### PR TITLE
Add a 'switch' and 'sample' command

### PR DESCRIPTION
**Pull request by cyberdelia**
Add a `switch` and `sample` commands that change switch or sample.
This is related to issue #10 but it's missing the `flag` command.

### CODE REVIEW

File: waffle/management/commands/flag.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    def handle(self, flag_name=None, *args, **options):
+        if not flag_name:
+            raise CommandError('You need to specify a flag name.')
+
+        try:
+            flag = Flag.objects.get(name=flag_name)
+            setattr(flag, attribute, value)
+            flag.save()
+        except Flag.DoesNotExist:
+            raise CommandError('This flag doesn\'t exists')
```
Comment (mwarkentin) [inline waffle/management/commands/flag.py]: Should be "This flag doesn\'t exist"

You could also do a get_or_create() instead

Comment (cyberdelia) [inline waffle/management/commands/flag.py]: I'm don't really want to create a new record from the command line without explicitly asking for it.

Comment (mwarkentin) [inline waffle/management/commands/flag.py]: For sure.. I ended up with this in my code:

```
make_option('--create',
            action='store_true',
            dest='create',
            default=False,
            help='If the flag doesn\'t exist, create it.'
        )

if options['create']:
            flag, created = Flag.objects.get_or_create(name=flag_name)
        else:
            try:
                flag = Flag.objects.get(name=flag_name)
            except Flag.DoesNotExist:
                raise CommandError('This flag doesn\'t exist')
```

Comment (jsocol) [inline waffle/management/commands/flag.py]: Nit, but when you have apostrophes in a string but no other quotes, it's usually preferable to use double quotes without backslashes. E.g. `"This flag doesn't exist."`. Fixed locally.

**Comment by jsocol** [comment]
I've made a bunch of style changes to make everything PEP8-happy and otherwise consistent (quote, comma styles, etc) with the rest of the code.

I don't love the current `--everyone/--deactivate` switches for Flags, because there's no way to get them back to a Null state, and it's not terribly clear how they're related. I'd like to do something like `--everyone [on/off/something]` instead.

**Comment by jsocol** [comment]
I added [two commits](https://github.com/jsocol/django-waffle/compare/management-commands) and will add documentation tomorrow. If either of you have feedback on the change, let me know!

**Comment by mwarkentin** [comment]
@jsocol Hey James, just wondering if there's any chance this will be making it in at some point? If the documentation is what's holding it up, I could take a swing at it when I have a bit of free time.

**Comment by jsocol** [comment]
Hey, I'm sorry about the delay. I've been trying to catch up on all these projects and PRs. I'll try to get to it today.

**Comment by mwarkentin** [comment]
Awesome, like I said, if there's anything I can help with, let me know.

**Comment by jsocol** [comment]
https://github.com/jsocol/django-waffle/compare/8c5788a...9c9aa5c

Thanks to you both!

**Pull request closed without merging**